components/layout/footer/site-info.tsx

```diff
@@ -6,14 +6,14 @@ export const SiteInfo = () => (
   <div>
     <CustomLink
       // TODO: Implementation
-      href="https://www.ssw.com.au/ssw/MenuMap.aspx"
+      href="/ssw/MenuMap.aspx"
     >
       SITEMAP
     </CustomLink>
     <Divider />
     <CustomLink
       // TODO: Implementation
-      href="https://www.ssw.com.au/ssw/HealthCheck"
+      href="/ssw/HealthCheck"
```

Comment:
I know you are adding a redirect for these, but I think you should fix the URL at the source as well.

Reply:
Also, there are TODOs in this code, please link them to a PBI

Reply:
> I know you are adding a redirect for these, but I think you should fix the URL at the source as well.

Done - I've fixed the links at the source

> Also, there are TODOs in this code, please link them to a PBI

Done - I've added links to the pbis for implementing these page